nflow-engine/src/main/java/io/nflow/engine/workflow/curated/MaintenanceWorkflowStarter.java

```diff
@@ -0,0 +1,46 @@
+package io.nflow.engine.workflow.curated;
+
+import io.nflow.engine.service.MaintenanceConfiguration;
+import io.nflow.engine.service.WorkflowInstanceService;
+import io.nflow.engine.workflow.instance.WorkflowInstance;
+import org.springframework.context.event.ContextStartedEvent;
+import org.springframework.context.event.EventListener;
+import org.springframework.core.env.Environment;
+import org.springframework.stereotype.Component;
+
+import static org.joda.time.Period.days;
+import static org.joda.time.Period.years;
+
+@Component
+public class MaintenanceWorkflowStarter {
+  protected final WorkflowInstanceService instanceService;
+  protected final boolean insertOnStartup;
+
+  public MaintenanceWorkflowStarter(Environment env, WorkflowInstanceService instanceService) {
+    this.instanceService = instanceService;
+    this.insertOnStartup = env.getProperty("nflow.maintenance.start", Boolean.class, false);
+  }
+
+  protected String getDefaultCronSchedule() {
```

Comment:
rename to getCronSchedule (I suppose this can be overridden, and then it is not the default anymore)

Reply:
I used the term default since this only affects the initial time the workflow engine starts. After that the configuration is through the database

Reply:
I see. Then it makes sense. Or should it be getInitialCronSchedule then?